addons/colorblind/addon.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "Color blindness accommodations",
+  "description": "A few features to help make Scratch more accessible to colorblind users.",
+  "credits": [
+    {
+      "name": "DNin01",
+      "link": "https://github.com/DNin01"
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "bold-links.css",
+      "matches": ["https://scratch.mit.edu/*"],
+      "if": {
+        "settings": { "links": "bold" }
+      }
+    },
+    {
+      "url": "underline-links.css",
+      "matches": ["https://scratch.mit.edu/*"],
+      "if": {
+        "settings": { "links": "underline" }
+      }
+    },
+    {
+      "url": "contrast-switch.css",
+      "matches": ["projects"],
+      "if": {
+        "settings": { "contrast-switch": true }
+      }
+    }
+  ],
+  "settings": [
+    {
+      "name": "Website: Link style",
+      "id": "links",
+      "type": "select",
+      "potentialValues": [
+        {
+          "id": "none",
+          "name": "Default"
+        },
+        {
+          "id": "bold",
+          "name": "Bold"
+        },
+        {
+          "id": "underline",
+          "name": "Underlined"
+        }
+      ],
+      "default": "bold"
+    },
+    {
+      "name": "Underline style",
+      "id": "underline-style",
+      "type": "select",
+      "potentialValues": [
+        {
+          "id": "solid",
+          "name": "Solid"
+        },
+        {
+          "id": "dashed",
+          "name": "Dashed"
+        },
+        {
+          "id": "dotted",
+          "name": "Dotted"
+        }
+      ],
+      "default": "solid",
+      "if": {
+        "settings": { "links": "underline" }
+      }
+    },
+    {
+      "name": "Editor: High contrast switches",
+      "id": "contrast-switch",
+      "type": "boolean",
+      "default": false
+    }
+  ],
+  "dynamicEnable": true,
+  "dynamicDisable": true,
+  "updateUserstylesOnSettingsChange": true,
+  "versionAdded": "1.34.0",
+  "tags": ["theme"],
```

Comment:
Seems like I have to either specify that this is a website theme or an editor theme. Have we ever made an addon that applies to both the editor and the website?

Reply:
Yes, but not often and I think all of them have been split.